sdk/docs/core/CHANGELOG.md

```diff
@@ -2,10 +2,22 @@
 
 
 ## 1.0.0-preview.4 (Unreleased)
-
+- API breaking changes:
+  - `az_context`:
+    - `key` and `value` are `const`.
+    - `az_context_with_expiration()` is renamed to `az_context_create_with_expiration()`.
+    - `az_context_with_value()` is renamed to `az_context_create_with_value()`.
+    - `az_context_app` is renamed to `az_context_application`.
+  - `az_credential_client_secret_init()` now takes fourth parameter, `authority`.
+  - `az_http_policy_retry_options`:
+    - `status_codes` now should be terminated by `AZ_HTTP_STATUS_CODE_END_OF_LIST`.
+    - `max_retries` is now `int32_t` instead of `int16_t`.
+  - `az_config.h`:
+    -  `AZ_HTTP_REQUEST_URL_BUF_SIZE` renamed to `AZ_HTTP_REQUEST_URL_BUFFER_SIZE`.
+    - `AZ_HTTP_REQUEST_BODY_BUF_SIZE` renamed to `AZ_HTTP_REQUEST_BODY_BUFFER_SIZE`.
+    - `AZ_LOG_MSG_BUF_SIZE` renamed to `AZ_LOG_MESSAGE_BUFFER_SIZE`.
 
 ## 1.0.0-preview.3 (2020-07-20)
-
```

Comment:
Leave this new line please. The markdown linter suggested it. See https://github.com/Azure/azure-sdk-for-c/pull/959 for more info.